### PR TITLE
Add introduction section

### PR DESCRIPTION
**Pull request by ayaderaghul**
- Add navigation bar
- Add introduction part, with text and background
- Add responsive style so that it works on both mobile and desktop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://google.com)
+- Coming soon..
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (wayungi) [inline README.md]: demo


- [x] If your project is not deployed, kindly delete this section 👍🏾

**Review by wayungi — COMMENTED** [review]
Hi @ayaderaghul , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [x] You have submitted a project that has an unprofessional readme

- [x] Please delete all instructions in your readme. instructions beign with `>` and the text appear grayed out in the readme when viewed from github
demo

- [x] I also suggest that when you choose to edit these lines, please delete the `>` at their begining

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ayaderaghul** [comment]
@wayungi 
Dear Francis,
I have made the changes in the README accordingly, before I request another review, can you please have a look?
I would appreciate that a lot, thanks

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <div class="clock">9:41</div>
+            <div class="icons">
+                <i class="fa-solid fa-signal"></i>
+                <i class="fa-solid fa-wifi"></i>
+                <i class="fa-solid fa-battery-full"></i>
+            </div>
+        </div>
+    
+        <nav class="logo">
+            <div class="mylogo">My Logo</div>
```
Comment (AlphaNtihinduka) [inline index.html]: - [x] I suggest that you consider having `a` tag for the logo. I you notice most websites have the logo, which when clicked redirects to the home page of the application. In addition, this is for three reasons: convention, semantics, and user experience(UX) ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello @ayaderaghul 🙋‍♂️
Good job working on your portifolio setup. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Matching the Figma design ✔️ 
- Proper use of semantic tagss ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

- _Kindly consider the inline review comments ☺️._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**